src/stories/containers/auth/login/login.tsx

```diff
@@ -1,15 +1,19 @@
 import styled from '@emotion/styled';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useThemeContext } from '../../../../core/context/ThemeContext';
 import { CustomButton } from '../../../components/custom-button/custom-button';
 import TextInput from '../../../components/text-input/text-input';
 import { useLoginMvvm } from './login.mvvm';
 import Image from 'next/image';
 
 export default () => {
-  const { form: formLogic, loading, error, clearErrors } = useLoginMvvm();
+  const { form: formLogic, loading, error, clearErrors, isMobile, isTable, clearCredentials } = useLoginMvvm();
   const { isLight } = useThemeContext();
 
+  useEffect(() => {
+    clearCredentials && clearCredentials();
+  }, []);
```

Comment:
this logic should be moved to the mvvm hook

Reply:
You are right